doc/syntax/syntax_en.tex

```diff
@@ -444,9 +445,9 @@ \section{Metadata declaration}
   \\
   State transitions declaration & \begin{catala}
     ```catala
-    internal var1 content ...
-      state before
-      state after
+      internal var1 content ...
+        state before
+        state after
```

Comment:
Don't you put the new syntax for toplevel definitions here ?

Reply:
I will, but I'd prefer to wait until we agree on a stable version — I am assuming undocumenting the syntax implies it's experimental for now :)

Reply:
Ok, I'll try to not forget this :)